packages/css-framework/src/components/_date-picker.scss

```diff
@@ -231,15 +232,17 @@
 }
 
 .rn-date-picker__day-grid {
-  display: grid;
-  grid-template-columns: repeat(7, 1fr);
-  justify-content: center;
+  display: flex;
+  flex-wrap: wrap;
+
+  * {
+    width: calc(100% / 7) %;
```

Comment:
I get a SASS error from this - it seems to evaluate it as an expression I think it see's the % as modulus operator.

Maybe we able to be more explicit about selector instead of using the wildcard (*)?